tests/docs/smoke-all/2024/07/18/10324.qmd

```diff
@@ -0,0 +1,56 @@
+---
+format: pdf
+keep-tex: true
+_quarto:
+  tests:
+    latex:
+      ensureFileRegexMatches:
+        - ['\\subcaption{\\label{tbl-example-2}Pressure}']
+        - []
+---
+
+::: {#tbl-example .cell layout-ncol="2" tbl-cap='Example' tbl-subcap='["Cars","Pressure"]'}
+
+```{.r .cell-code}
+library(tinytable)
+tt(head(cars)) |> theme_tt("tabular") |> 
```

Comment:
Just a reminder that `theme_tt("tabular")` only works in the development version of `tinytable`. Hopefully this doesn't break your tests.

Reply:
See above :)